OWNERS

```diff
@@ -1,12 +1,8 @@
 reviewers:
-  - jpower432
```

Comment:
Hi @ncdc, 

Since this CVE is going to be backported for 4.14, 4.13, 4.12, 4.11 and 4.10 we thought it was a good opportunity to change the OWNERS file instead of opening 5 PRs only for that change and spending more time of staff engineers.

If you want we can remove it from the 5 PRs and create 5 new ones, there is not problem on our side.

Reply:
That's fine, thanks.